x-pack/plugins/global_search_providers/kibana.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "globalSearchProviders",
```

Comment:
SGTM. Once #68811 is merged, we should group this plugin with the global_search plugin in the same directory.

Reply:
Was thinking about this more and maybe this should live in the same plugin. Where do we expect the search UI code to go? Since it needs to be under the Elastic License, it must be in a plugin. I see two options:
- A 3rd plugin, like `global_search_ui` that depends on `global_search` and `global_search_providers`
- Putting everything in one plugin

cc @myasonik

Reply:
🤷‍♂️ I'm a terrible person to ask in this case. I've got little context on the tradeoffs of this... Either options _seems_ fine.

I'd maybe err towards separate plugins in case we ever want to experiment with different search strategies, it might be easier to swap between two separate plugins rather than handling it all in one.

But, really, I'm just guessing at a bunch of this so other opinions encouraged.

Reply:
> Was thinking about this more and maybe this should live in the same plugin. Where do we expect the search UI code to go

I was thinking of something like `global_search_bar` or something like that tbh.

I added a new plugin for the GS providers basically only because I could (tm) and it was maybe a slightly better isolation. But I have no strong feeling toward this. I can move the providers into the `global_search` plugin if we want to.

For the `searchbar` / `search_ui`, I still thinks it makes more sense to have it as a separate plugin, as it's supposed to be 'just' a consumer of the `GS` API. But here too, I will follow the majority if we think it's better to have everything in a single plugin.

Reply:
Let's just go with separate plugins for everything and then group them in a single directory once #68811 merges.